app/src/main/java/net/squanchy/speaker/view/SpeakersView.java

```diff
@@ -0,0 +1,44 @@
+package net.squanchy.speaker.view;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+
+import java.util.List;
+
+import net.squanchy.speaker.Speaker;
+
+public class SpeakersView extends RecyclerView {
+
+    private SpeakerAdapter adapter;
+
+    public SpeakersView(Context context) {
```

Comment:
You don't need this ctor, and you're missing the 3-param `Context context, AttributeSet attrs, int defStyleAttr` ctor which is used by Android when inflating. ⚠️